src/utils.js

```diff
@@ -26,6 +26,15 @@ export function humanizeTimestamp(timestamp, tDateTimeParser) {
   return time.from(now);
 }
 
+// https://reactnative.dev/docs/linking
+export const sanitizeUrlForLinking = (url) => {
+  if (url.indexOf('http://') === -1 && url.indexOf('https://') === -1) {
```

Comment:
pretty much the same thing
```suggestion
  if (url.indexOf('http') === -1) {
```

Reply:
alright updated it with regex

Reply:
Thanks alot @virdesai :)